public/data/food/processes/builder.json

```diff
@@ -31,7 +31,9 @@
       "bvi": 0.2271,
       "etf-c": 1.29211471562,
       "htc-c": 1.577471183e-10,
-      "htn-c": 2.46769729382e-09
+      "htn-c": 2.46769729382e-09,
+      "pef": 0.0,
+      "ecs": 0.0
```

Comment:
Hmm why do we suddenly need these aggregated impacts as I think we decided they should always be computed in Elm. Have I missed something?

Reply:
Yes, so, about this: at the moment, the PEF scores are computed at the "exporting ciqual products" step in `export_ciqual.py` **for the food explorer**. But in any case, they are computed at the decoding step (so we actually don't use the decoded aggregated scores that are present for the food explorer).

I'm going to also modify the `export_ciqual.py` script to avoid compute the PEF score, if that's ok with @ccomb and @paulboosz ?